website/docs/r/ssl_certificate.html.markdown

```diff
@@ -0,0 +1,155 @@
+* `orderApproverEmailAddress` - (Required, string) The email of approver to approve ssl certificate request.
+* `org_addressLine1` - (Required, string) The address of organization who is requesting for ssl certificate.
+* `org_addressLine2` - (optional, string) The address of organization who is requesting for ssl certificate.
+* `org_city` - (Required, string) The city of organization which is requesting for ssl certificate .
+* `org_postalCode` - (Required, integer) The postal code for the city of organization.
+* `org_state` - (Required, string) The two letter state code of organization who is requesting for ssl certificate. Allowed value for country which doesn't have states is `OT`.
+* `org_countryCode` - (Required, string) The two letter country code of organization.
+* `org_organizationName` - (Required, string) Name of organization.
+* `tech_addressLine1` - (Required, string) The address for technical contact.
+* `tech_addressLine2` - (optional, string) The address for technical contact.
+* `tech_city` - (Required, string) The city for technical contact.
+* `tech_postalCode` - (Required, integer) The postal code for technical contact.
+* `tech_state` - (Required, string) The two letter state code of technical contact. Allowed value for country which doesn't have states is `OT`.
+* `tech_countryCode` - (Required, string) The two letter country code for technical contact.
+* `tech_organizationName` - (Required, string) Name of organization for technical contact.
+* `tech_firstName` - (Required, string) The first name for technical contact.
+* `tech_lastName` - (Required, string) The last name for technical contact.
+* `tech_title` - (Required, string) The title for for technical contact.
+* `tech_emailAddress` -(Required, string) email address for technical contact.
+* `admin_addressLine1` - (Optional, string) The address for administrative contact.
+* `admin_addressLine2` - (optional, string) The address for administrative contact.
+* `admin_city` - (Optional, string) The city for administrative contact.
+* `admin_postalCode` - (Optional, integer) The postal code for administrative contact.
+* `admin_state` - (Optional, string) The two letter state code of administrative contact. Allowed value for country which doesn't have states is `OT`.
+* `admin_countryCode` - (Optional, string) The two letter country code for administrative contact.
+* `admin_organizationName` - (Optional, string) Name of organization for administrative contact.
+* `admin_firstName` - (Optional, string) The first name for administrative contact.
+* `admin_lastName` - (Optional, string) The last name for administrative contact.
+* `admin_title` - (Optional, string) The title for for administrative contact.
+* `admin_emailAddress` -(Optional, string) email address for administrative contact.
+* `billing_addressLine1` - (Optional, string) The address for billing contact.
```

Comment:
Add all high level attributes organization_information and define the inner attributes as nested attributes...for all attributes(techincal,administrative..so on)
https://ibm-cloud.github.io/tf-ibm-docs/v0.16.1/r/iam_user_policy.html

Reply:
Add Attribute Reference section also

Reply:
Done. please have a look.